ush/run_fv3.ksh

```diff
@@ -0,0 +1,39 @@
+#!/bin/ksh --login
+ 
+# Set the queueing options 
+#SBATCH --nodes=60 --ntasks-per-node=4   ## refer to the run_fcst task section in the workflow xml
+#SBATCH -t 6:40:00
+#SBATCH -A nrtrr
+# #SBATCH -qos debug
+#SBATCH -J fv3_fcst
+#SBATCH --partition=sjet  ## check which jet partition was used from the run_fcst log file in the cycle that needs to be repeated
+ 
+set -x
+#-------------------------------------------
+# point to the source code for executable and
+# for loading the same modules used in the run that needs to be repeated
+#-------------------------------------------
+SRCDIR=/WHERE-YOUR-ufs-srweather-app-CODE-IS/dev4-ufs-srweather-app
+module purge
+module use $SRCDIR/regional_workflow/modulefiles/tasks/jet
+module load run_fcst
+ 
+FV3_EXE=$SRCDIR/exec/fv3_gfs.x
+
+#-------------------------------------------
+# the workdir is a hard copy of the cycle that needs to be repeated, for example, /lfs4/BMC/nrtrr/NCO_dirs/stmp/tmpnwprd/RRFS_dev4/2021010612
+# remove the IC/LBC source (HRRRX/, RAPX/), forecast files (dynf*.nc, phyf*.nc), log files (logf*), post and restart files (if applicable) 
+# must keep all the fix files and INPUT/ 
+#-------------------------------------------
+workdir=/WEHER-YOUR-TEST-DIR-IS/2021010612
+cd $workdir
+rm -f fv3_gfs.x
+cp ${FV3_EXE} fv3_gfs.x
```

Comment:
Are we sure we want this script to handle the exe? I thought the point was that it would just run whatever is in the run directory. Maybe the user wants the original exe. Maybe they want to link in a new one. Letting the script do this could end up being pretty frustrating.

Reply:
The original run directory has no fv3.exe since it is handled by the xml. For this stand-alone script without the rocoto workflow, we need have the executable. And it points to where the source code is for the user.

Reply:
I didn't even realize that nuance. I really thought the executable had to be in the run directory and not just called from the run directory! 

Let's see what Joe et al think about this functionality.

Reply:
In the "develop" mode that Jeff/Gerard are proposing, hopefully, it will save the exe too, so we can determine reproducibility with ease.

Reply:
Let's not remove it here, then. Just in case we want to use the same exe.